src/testnet/airdrop.js

```diff
@@ -1,12 +1,61 @@
-import tokens from '@aragon/templates-tokens'
```

Comment:
I've removed this import and just inlined the contents so we can control them better.

Particularly, the import used the network's name rather than chain ID (and to be honest, that package is not particularly helpful).